### ISSUE TITLE
UX: Double Login buttons due to WebAuthn Passwordless Login

### ISSUE THREAD

**Issue by anibalsanchez** [joomla/joomla-cms#36060]
### Steps to reproduce the issue

By default, the **WebAuthn Passwordless Login** plugin is enabled by default. When there are no users using WebAuthn or any intention to use WebAuthn Passwordless.

The backend users are confused due to the double Login Buttons.

### Expected result

One Login button to the Login action. Or, at least, disable the **WebAuthn Passwordless Login** plugin.

### Actual result

Two Login buttons

### System information (as much as possible)

Joomla 4.0.4

### Additional comments

**Workaround**: manually disable WebAuthn Passwordless Login on new J4 installations.

**Comment by HLeithner** [comment]
WebAuthn should be more visible and explain why to use it instead of hiding it. my2c

**Comment by joeforjoomla** [comment]
99% of users won't use WebAuthn. Showing the button by default in the backend login is simply confusing. It should be treated as the 2FA plugin, enabled only when needed.

**Comment by anibalsanchez** [comment]
I agree that WebAuthn is important. But, duplicating the Login action button is a UX error.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36060">issues.joomla.org/tracker/joomla-cms/36060</a>.</sub>

**Comment by anibalsanchez** [comment]
BTW: I'm a WebAuthn user and I'm used to the product, but this is the first time I found a duplicated Login button and a **WebAuthn** button ever.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36060">issues.joomla.org/tracker/joomla-cms/36060</a>.</sub>

**Comment by Quy** [comment]
Duplicate #30905

**Comment by HLeithner** [comment]
> BTW: I'm a WebAuthn user and I'm used to the product, but this is the first time I found a duplicated Login button and a **WebAuthn** button ever.


In my opinion the login process should be more "guided" like google, only presenting you the login possibilities your "username" allows, for this we need a multi step ajax login.

**Comment by anibalsanchez** [comment]
> Duplicate #30905

The issue is an error. If you document the workaround and accept that the UX is right, then there is nothing else to say.

**Comment by simbus82** [comment]
> ### Steps to reproduce the issue
> By default, the **WebAuthn Passwordless Login** plugin is enabled by default. When there are no users using WebAuthn or any intention to use WebAuthn Passwordless.
> 
> The backend users are confused due to the double Login Buttons.
> 
> ### Expected result
> One Login button to the Login action. Or, at least, disable the **WebAuthn Passwordless Login** plugin.

Totally agree.
As important as it is (I'm already using it on all j4 sites, with my smartphone and fingerprint ... super cool!), it can't be activated by default without a previous "info" that inform the admin user of what to expect when he do the first logging into the backend.
For me:
- it should not be activated by default because it's not right to show the WebAuthn button on first use after Joomla installation, because no one has had the opportunity to set an authenticator in the backend yet.
- it should be always hidden, even if active, in case none of all users have activated an authenticator.

PS: for UX best practice it is a mistake to show two buttons with the same visual impact: maybe we can use a different color.

> In my opinion the login process should be more "guided" like google, only presenting you the login possibilities your "username" allows, for this we need a **multi step ajax login**.

Please... Not for an admin login page in an on-premise CMS.
A website is not an authentication provider like Google.
We already have a rather chaotic login screen, let's not add any more complications on the UX side!

**Comment by painsadyu** [comment]
[Friday Night Funkin](https://fridaynightfunkin.io/friday-night-funkin) is not what you think when looking at the first images of the poster. It's literally the funniest witty music I've ever played.

And the surprise is that it also has a storyline, not flying in and entering like other music games. No matter what the next steps are, in the beginning, I gave it an A+ because of the difference.

The story of the game is divided into several weeks, representing the encounters and challenges of the godfather for you. Each week there will be 3 different difficulty levels. The harder it is, the faster your rhythmic coordination as well as your speed of grasping the notes should be.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36060">issues.joomla.org/tracker/joomla-cms/36060</a>.</sub>

**Comment by Quy** [comment]
See PR #44228
